backend/cinema-service/cp-trace/catchpoint-config-qa.yml

```diff
@@ -0,0 +1,26 @@
+catchpoint:
```

Comment:
Remove env config file. There should be only one config file for production which might be `catchpoint-config.yml`